lib/windows.js

```diff
@@ -68,6 +73,44 @@ const kernel32 = new ffi.Library('kernel32', {
 	QueryFullProcessImageNameW: ['int', ['pointer', 'uint32', 'pointer', 'pointer']]
 });
 
+function getProcessPathAndName(processHandle) {
```

Comment:
Looking more closely at the code, I think it's enough to make this function only return the `processPath` as the name is just a `basename` call away.

Reply:
Resolved.